google/cloud/forseti/scanner/scanners/iap_scanner.py

```diff
@@ -329,14 +350,17 @@ def is_alternate_service(self, backend_service, backend_service2):
 
             network_port = self.instance_group_network_port(
                 backend_service, instance_group)
+            if not network_port:
+                continue
+
             for backend2 in backend_service2.backends:
                 instance_group2 = self.find_instance_group_by_url(
                     backend2.get('group'))
                 if not instance_group2:
                     continue
                 network_port2 = self.instance_group_network_port(
                     backend_service2, instance_group2)
-                if network_port != network_port2:
+                if not network_port2 or network_port != network_port2:
```

Comment:
Can you please break up the if conditions here into 2?  The code here are all standalone if's, and to me is also more clear.
```
if not network_port2:
    continue

if network_port != network_port2:
    continue